runtime/parser2/expression.go

```diff
@@ -330,8 +322,6 @@ func defineArrayExpression() {
 }
 
 func defineDictionaryExpression() {
-	leftBindingPowers[lexer.TokenBraceOpen] = 150
```

Comment:
`{` isn't used as a left denotation, e.g. we don't have something like `foo{bar...`, so this was wrong. 

Left binding powers for `(` and `[` above
(https://github.com/onflow/cadence/pull/82/files#diff-dc3156241c5da3fc88bf5e2220db331aR302 and
https://github.com/onflow/cadence/pull/82/files#diff-dc3156241c5da3fc88bf5e2220db331aR293) are still correct because they are used for function calls (`foo(...)`) and indexing (`foo[...]`), which will be added later